src/java.desktop/windows/native/libawt/windows/awt_Window.cpp

```diff
@@ -1396,6 +1396,15 @@ BOOL AwtWindow::UpdateInsets(jobject insets)
     RECT inside;
     int extraBottomInsets = 0;
 
+    OSVERSIONINFOEX info;
+    ZeroMemory(&info, sizeof(OSVERSIONINFOEX));
+    info.dwOSVersionInfoSize = sizeof(OSVERSIONINFOEX);
+    GetVersionEx((LPOSVERSIONINFO) &info);
+
+    // add extra padded border only when Win OS version is 6+
+    int extraPaddedBorderInsets = info.dwMajorVersion >= 6 ?
+     ::GetSystemMetrics(SM_CXPADDEDBORDER) : 0;
+
```

Comment:
The OS version check is added since **SM_CXPADDEDBORDER** metric is not supported on Win XP & 2000. If the current JDK Window target OS version is set to Windows Vista, probably this check is not required or redundant. Suggestions would be helpful as to whether to retain this check or not

![image](https://user-images.githubusercontent.com/95945681/173000377-4109fd09-f8e3-43a3-aa62-3d0c67d2ff56.png)
 
Screenshot source: [GetSystemMetrics Doc](https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-getsystemmetrics)

Reply:
You can look at the oficially supported versions of operating systems by different Java versions here: https://www.oracle.com/java/technologies/javase-subscription/documentation.html#sysconfig
Nothing even remotely relevant supports Windows XP.

Reply:
@azuev-java Thank you for clarifying and sharing the link. It's a good reference for future as well.